osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmapSet.cs

```diff
@@ -51,9 +51,7 @@ public partial class DrawableCarouselBeatmapSet : DrawableCarouselItem, IHasCont
         [Resolved]
         private IBindable<RulesetInfo> ruleset { get; set; } = null!;
 
-        public IEnumerable<DrawableCarouselItem> DrawableBeatmaps => beatmapContainer?.IsLoaded != true ? Enumerable.Empty<DrawableCarouselItem>() : beatmapContainer.AliveChildren;
-
-        private Container<DrawableCarouselItem>? beatmapContainer;
+        public Container<DrawableCarouselItem>? Beatmaps;
```

Comment:
I think I'd prefer this to be private and exposed as an `IReadOnlyList`.

Reply:
Just to confirm, this means using the `for` loop as you proposed?

Reply:
Yeah.